addons/im_livechat/models/mail_message.py

```diff
@@ -41,27 +41,20 @@ def _to_store(self, store: Store, **kwargs):
                     .sudo()
                     .search([("mail_message_id", "=", message.id)], limit=1)
                 )
-                if chatbot_message.script_step_id:
+                if step := chatbot_message.script_step_id:
+                    step_data = {
+                        "id": (step.id, channel.id),
+                        "message": Store.one(message, only_id=True),
```

Comment:
Message params should ideally (can be done later) be given here rather than as a separate add below.
We might need to add support for *args forwarding in `one` though.